scripts/steps/build-api.js

```diff
@@ -110,6 +104,49 @@ const UNWRAP_UTILITIES = new Set([
   'Readonly'
 ])
 
+const EXCLUDED_SUBMENU_KINDS = new Set(['Type alias', 'Interface', 'Namespace'])
+const EXCLUDED_TREE_KINDS = new Set(['Namespace'])
+
+const GROUPS = {
+  Actions: 'Actions',
+  Client: 'Client',
+  Core: 'Core',
+  Other: 'Other',
+  React: 'React',
+  Server: 'Server',
+  State: 'State',
+  Tests: 'Tests',
+  Types: 'Types',
+  Vue: 'Vue'
+}
+
+// More specific group conditions should be placed higher
+const GROUPS_CONDITION = {
+  [GROUPS.Types]: node =>
```

Comment:
The was no `Types` group in the task

Reply:
Should types be in the group "other"?

Reply:
Yeap. Most of them are not so important.